src/lualib/Promise.ts

```diff
@@ -0,0 +1,335 @@
+declare function error( value:string, level?:number ): never;
+const logError: any = print;
```

Comment:
In `inline` lib mode all variables in lib's scope would be included as they are, so they should be prepended with `____` to avoid potential conflicts.